src/Util/GeometryUtil/GeometryUtil.js

```diff
@@ -9,15 +9,18 @@ import {
   buffer,
   centroid,
   difference,
-  getCoords,
   intersect,
-  multiLineString,
   polygonize,
   polygonToLine,
-  segmentEach,
   union
 } from '@turf/turf';
```

Comment:
Why don't we have separate imports for the other turf modules like `@turf/buffer` here (and maybe in other modules)?

Reply:
i think this will be fixed in upcoming commit to this PR

Reply:
yeah I am working on getting @weskamm's 
closed  <a href="https://github.com/terrestris/react-geo/pull/661">pull request</a> integrated in this PR.

Reply:
:+1: